resources/views/dashboard.blade.php

```diff
@@ -54,6 +44,9 @@
                   //$("#expForm").validate();
                   </script>
               </div>
+              <div>
+                <table class='table'><thead><tr><th>OWED</th><th>OWES</th><th>TTL</th><th>BAL</th></tr></thead><tbody><tr><td class='text-success'><strong>+0</strong></td><td class='text-danger'><strong>-0</strong></td><td>0</td><td>0</td></tr></tbody>
```

Comment:
Im not enforcing these changes on this branch btw. This could be done later in our refactoring stage. We can leave this as a TODO